infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -506,64 +510,76 @@ private void trimLogFile(String filePath, Set<Long> pendingTrim) throws IOExcept
         writeChannels.remove(filePath);
     }
 
+    /** Scan all entries, updating the LogUnit structures and verifying the log file integrity.
+     *
+     * @param filePath  The log file (segment) to scan
+     */
+    private void scanAndVerifyEntries(String filePath) throws IOException {
+        FileChannel fc = getChannel(filePath, true);
+        try (MappedByteBufInputStream stream = new MappedByteBufInputStream(fc)) {
+            Metadata headerMetadata = Metadata.parseFrom(stream.limited(METADATA_SIZE));
+            LogHeader header = LogHeader.parseFrom(stream.limited(headerMetadata.getLength()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "header" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)